app/assets/javascripts/create_idea.js

```diff
@@ -0,0 +1,24 @@
+function createIdea() {
+  var title = $('#title').val();
+  var body = $('#body').val();
+  var ideaParams = { idea: { title: title, body: body } };
+  $.ajax({
+    type: 'POST',
+    url:  '/api/v1/ideas',
+    data: ideaParams,
+    success: function(idea) {
+      prependIdea(idea);
+      clearTextFields();
+    },
+    dataType: 'JSON'
+  });
+}
+
+function clearTextFields() {
+   $('#title').val('');
```

Comment:
you could define vars for #title and #body since you find the selector twice. same with all ideas